Don't insert * after new line in multi-line comment if previous line … (#3392)
geany/geany: doc/geany.txt

      *

    on the next line with the correct indentation based on the previous line,
    as long as the multi-line is not closed by ``*/``. If the previous line

>     has no ``*`` prefix, the new line will not be changed.

Reviewer: Suggest "the new line will not be changed" s/b "no ``*`` will be added to the new line" just to be explicit

Author: ```suggestion
    has no ``*`` prefix, no ``*`` will be added to the new line.
```
